### PR TITLE
Updates to make command work

### PR DESCRIPTION
**Pull request by SeanFeldman**
1. Front can send now command to backend
2. Replaced system.Version with Nuget`s SymaticVersion (might need to review this, but system.Version has a flaw when parsed)
3. Standardized Transport connection string to Environment variable

### CODE REVIEW

File: APIComparer.Backend/APIComparer.Backend.csproj [hunk 1]
```diff
@@ -89,9 +89,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\NServiceBus.5.2.0\lib\net45\NServiceBus.Core.dll</HintPath>
     </Reference>
-    <Reference Include="NuGet.Core, Version=2.8.50926.602, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="NuGet.Core, Version=2.8.50506.491, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (andreasohlund) [inline APIComparer.Backend/APIComparer.Backend.csproj]: Why did we need to downgrade?

File: APIComparer.Contracts/CompareNugetPackage.cs [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
             PackageId = packageId;
             RightVersion = rightVersion;
             LeftVersion = leftVersion;
         }
 
-        public string PackageId { get; private set; }
-        public Version LeftVersion { get; private set; }
-        public Version RightVersion { get; private set; }
+        public string PackageId { get; set; }
+        public SemanticVersion LeftVersion { get; set; }
```
Comment (andreasohlund) [inline APIComparer.Contracts/CompareNugetPackage.cs]: Not sure its a good idea to tie the message contract to a nuget type?

How about just a string for now?

File: APIComparer.Backend/APIComparer.Backend.csproj (hunk 1 above)
Comment (SeanFeldman) [inline APIComparer.Backend/APIComparer.Backend.csproj]: We had 2 versions, both 2.8.3 and 2.8.2. Thought I have introduced 2.8.3, so went with 2.8.2

File: APIComparer.Contracts/CompareNugetPackage.cs (hunk 2 above)
Comment (SeanFeldman) [inline APIComparer.Contracts/CompareNugetPackage.cs]: @SimonCropp suggested NuGet version. Would need to see what's that first and then fall back to string if not an option.

Comment (danielmarbach) [inline APIComparer.Contracts/CompareNugetPackage.cs]: In the first call we agree on not using SemanticVersion of nuget. If this doesn't have restrictions when we analyze non semantic version compliant packages I'm OK with that change but I agree with @andreasohlund that we shouldn't tie the message contract to the nuget type.

Comment (SeanFeldman) [inline APIComparer.Contracts/CompareNugetPackage.cs]: Apology, I was not part of the 1st call.
NuGet`s `SematicVersion` can work with `major`, `minor`, `build`, and `revision`. Doesn't that make it compatible with .NET `Version`?

Comment (danielmarbach) [inline APIComparer.Contracts/CompareNugetPackage.cs]: Good. Then I’ll shut up :D

Comment (SeanFeldman) [inline APIComparer.Contracts/CompareNugetPackage.cs]: No need to shut up, you were just expressing a concern.
I've looked it up with reflector and it looks like behind the scenes `SemanticVersion` is storing info in `Version` :)

File: APIComparer.Backend/APIComparer.Backend.csproj (hunk 1 above)
Comment (SeanFeldman) [inline APIComparer.Backend/APIComparer.Backend.csproj]: @andreasohlund updated all to 2.8.3

**Pull request merged**